### ISSUE TITLE
informations about the App

### ISSUE THREAD

**Issue by trkmh19** [AymenSe/INTTIC-Semester-rate-java-oop#2]
We need more information about the use cases and specifications of the APP.

**Comment by AymenSe** [comment]
This app allows you to calculate the semester rate for INTTIC student. Since I lack a real structure for this app. you can imagine how we can put everything together to achieve our end.
I am going to update the README file to provide more information about the APP.
Feel free to share your ideas with us.

**[Check README](https://github.com/AymenSe/INTTIC-Semester-rate-java-oop/blob/main/README.md)**

**Comment by trkmh19** [comment]
great as a beginning, my vision is to build ( for now ) the most minimalistic app architecture and with the minimum of functionalities, then with time, we can improve things, and make more features and accumulate more and more things

**Comment by AymenSe** [comment]
OF COURSE. That's okay for now. But we have to design the MOST important classes and define them. We need to visualize how classes interact with each other. 
I am trying to design the app in "lucid.app", [you can contribute here](https://lucid.app/lucidchart/invitations/accept/66679806-a171-4e64-8a1a-8bbe274848d1).
